features/shipping/managing_shipping_methods/denying_access_to_shipping_methods_for_unauthorized_users.feature

```diff
@@ -0,0 +1,56 @@
+@managing_shipping_methods
+Feature: Denying access to shipping methods for unauthorized users
+    In order to denies access for unauthorized users
+    As an Visitor
+    I don't want to access to manage shipping methods
+
+    Background:
+        Given the store operates on a channel named "Web-US" in "USD" currency
+        And the store is available in "English (United States)"
+        And the store has a zone "United States" with code "US"
+        And the store allows shipping with "UPS Carrier" identified by "UPS_CARRIER"
+
+    @api
+    Scenario: Trying to add a new shipping method as a unauthorized user
+        When I want to create a new shipping method
+        And I specify its code as "FED_EX_CARRIER"
+        And I specify its position as 0
+        And I name it "FedEx Carrier" in "English (United States)"
+        And I define it for the zone named "United States"
+        And I choose "Flat rate per shipment" calculator
+        And I specify its amount as 50 for "Web-US" channel
```

Comment:
Can you merge it to one step?

Reply:
Its not easy because, If I try do it, All needed information I must hardcode in context, That step can confuse in future when someone try reuse it

Reply:
This information does not have to be included in the step itself. "When I try to create a new shipping method with valid data" or "When I try to create a new shipping method named "FedEx Carrier".